app/controllers/items_controller.rb

```diff
@@ -0,0 +1,76 @@
+require 'net/http'
```

Comment:
I don't think you need this...should be automatically included. Try it without (you may not need this up here)